geozero/tests/svg.rs

```diff
@@ -23,6 +19,12 @@ fn json_to_svg() -> Result<()> {
     Ok(())
 }
 
+/*
```

Comment:
If you want to keep this, can you leave a comment for us-in-the-future explaining why it's here but commented out?